src/validation.cpp

```diff
@@ -60,6 +60,8 @@
 
 CChainState g_chainstate;
 
+CChainState* ChainstateActive() { return &g_chainstate; }
```

Comment:
In commit "refactoring: introduce ChainstateActive()" (612d1b5df556eb70eb5fb670d166a1d7927e6efc)

This is returning a pointer, but none of the places calling it are checking null, so it seems like it'd be better to return a reference instead. Also would make `ChainstateActive()` more consistent with `ChainActive()`